src/CiviliansAccountsMicroservice/Civilians.Core/ViewModels/Civilians/LoginViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Civilians.Core.ViewModels.Civilians
+{
+    /// <summary>
+    /// TODO: Add detailed validation
+    /// </summary>
+
+    public class LoginViewModel
```

Comment:
Do you have model and viewModel on one layer?

Reply:
I choose viewmodels location depending on their place of use. 
If a viewmodel needs to be placed as a parameter in the repository method i'll put it right on the same layer.

Reply:
As a parameter to repository method you should pass entity and do everything with it

Reply:
Done